.buildkite/pipeline.yaml

```diff
@@ -45,12 +45,12 @@ steps:
 
   - wait
 
-  - name: ":seedling: Publish: ${KAIZEN_DOMAIN_NAME}"
+  - name: ":books: Publish Storybook: ${KAIZEN_DOMAIN_NAME}"
     key: "publish-step"
     command: ".buildkite/scripts/publish.sh"
     timeout_in_minutes: 15
     # The canary branch is not able to update dev.cultureamp.design
-    branches: "!canary"
+    if: build.branch == "main" && (build.author.name != "github-actions[bot]" || build.pull_request.labels includes "force publish storybook")
```

Comment:
This isn't a GH action though? I don't think it has access to these?

Reply:
Yeah Buildkite actually gets some of the GH PR details
https://buildkite.com/docs/pipelines/conditionals

Reply:
Ohhh snazzy.

I don't think your labels condition works though. The `main` branch isn't a PR 😅

Reply:
Oh good point...